extensions/wikia/AnalyticsEngine/AnalyticsProviderNielsen.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+class AnalyticsProviderNielsen implements iAnalyticsProvider {
+
+	private static $apid = 'T26086A07-C7FB-4124-A679-8AC404198BA7';
+	private static $clientId = 'Wikia';
+	private static $libraryUrl = 'http://secure-dcr-cert.imrworldwide.com/novms/js/2/ggcmb500.js';
+
+	function getSetupHtml( $params=array() ) {
```

Comment:
Meh :stuck_out_tongue_closed_eyes:. Just to make it consistent with other providers.